package/rpi-firmware/rpi-firmware.mk

```diff
@@ -5,9 +5,9 @@
 ################################################################################
 
 ifneq ($(BR2_PACKAGE_LIBDRM),y)
-RPI_FIRMWARE_VERSION = 01ecfd2ba2b7cf3a2f4aa75ada895ee4a3e729f5
+RPI_FIRMWARE_VERSION = 52185fddc0a9a28472fb3387dbed1a5466e11242
 else
-RPI_FIRMWARE_VERSION = d016a6eb01c8c7326a89cb42809fed2a21525de5
+RPI_FIRMWARE_VERSION = 52185fddc0a9a28472fb3387dbed1a5466e11242
```

Comment:
This is dangerous, as you're changing the package version used for every build. There's probably a reason for using two different versions here depending on whether LIBDRM is being used (maybe bugs or required features or whatever) and you're ignoring it, which may break existent configurations. If you need a different version of the firmware just for a new rpi, the config should probably reflect that in the name and use the new firmware version only for that config. Or even better, add an option to the rpi firmware package configuration in menuconfig to choose between the old and new version.